pkg/cluster/internal/create/actions/kubeadminit/init.go

```diff
@@ -114,43 +112,26 @@ func (a *action) Execute(ctx *actions.ActionContext) error {
 	return nil
 }
 
-// matches kubeconfig server entry like:
-//    server: https://172.17.0.2:6443
-// which we rewrite to:
-//    server: https://localhost:$PORT
-var serverAddressRE = regexp.MustCompile(`^(\s+server:) https://.*:\d+$`)
-
 // writeKubeConfig writes a fixed KUBECONFIG to dest
 // this should only be called on a control plane node
-// While copyng to the host machine the control plane address
+// While copying to the host machine the control plane address
 // is replaced with local host and the control plane port with
 // a randomly generated port reserved during node creation.
-func writeKubeConfig(n *nodes.Node, dest string, hostPort int32) error {
+func writeKubeConfig(n *nodes.Node, dest string, hostPort int32, clusterName string) error {
 	cmd := n.Command("cat", "/etc/kubernetes/admin.conf")
-	lines, err := exec.CombinedOutputLines(cmd)
+	buff, err := exec.Output(cmd)
```

Comment:
(I'm not too familiar here, so please excuse my question if it's a stupid one!) why the switch from CombinedOutputLines to Output?